### ISSUE TITLE
Combine the export feature with scan feature

### ISSUE THREAD

**Issue by mikegunning** [TokyoYoshida/ExampleOfiOSLiDAR#12]
Thanks for the examples. 

Is it possible to combine the 2 feature to export the MDLMesh with the captured image from the camera to have a .obj model?

**Comment by TokyoYoshida** [comment]
@mikegunning 
In theory it is possible. However, on iOS14, when I try to convert SCNGeometry to MDLMesh and export it to .obj, it seems to crash. This topic was also mentioned on [Stack Overflow](https://stackoverflow.com/questions/64179922/export-arscnfacegeometry-wrapped-in-mdlasset-fail-on-ios-14-with-exc-bad-access).

**Comment by mikegunning** [comment]
Yes I noticed this too playing with your code. MDLMesh also removes the texture regardless.

 In the end I settled with writing the scene [with this](https://developer.apple.com/documentation/scenekit/scnscene/1523577-write).

Unfortunately the only format that allows both mesh AND texture is .scn which is pretty unusable outside the Apple eco system.

**Comment by SashaBes** [comment]
@mikegunning 
Good evening,
any chance you could share the project or your code adjustments that allow the .scn export? 
Thanks in advance, 
Aleksandra

**Comment by farhodyusupov** [comment]
hi, is there any news about this topic?
